src/main/java/gov/cabinetofice/gapuserservice/dto/UpdateDepartmentReqDto.java

```diff
@@ -0,0 +1,18 @@
+package gov.cabinetofice.gapuserservice.dto;
+
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+@Data
+@Getter
+@Setter
+public class UpdateDepartmentReqDto {
+    @NotBlank(message = "Enter a department name.")
+    private String departmentName;
```

Comment:
Likely need a max var char count so we don't exceed the DBs size for this column